### PR TITLE
firmware update to be released with 3.4

### PR DESCRIPTION
**Pull request by ctech4285**

### CODE REVIEW

File: common/source/docs/common-electro-permanent-magnet-V3.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ Firmware setup
  
 .. note::
 
-    A pull request has been issued to include the changes in the next APM release. This documentation will be update when the request has been accepted. If this annoys you please write a comment `here <https://github.com/ArduPilot/ardupilot/pull/4361>`__
+    The firmware update is in maser and do, to be released with Copter-3.4A.
```
Comment (hamishwillee) [inline common/source/docs/common-electro-permanent-magnet-V3.rst]: Spelling errors. Should be 

    The firmware update is in master and is to be released with Copter-3.4A.

File: common/source/docs/common-electro-permanent-magnet-V3.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ Objects that it grips onto should be metal or have the included `thin steel targ
 attached to them.
 
 The device accepts a standard PWM servo input or UAVCAN and uses a 5V power supply.  
-Significant amounts of power are only required for about about 1 second when it first grips or releases the cargo.
+Significant amounts of power is only required for about 1 second when it first grips or releases the cargo.
```
Comment (hamishwillee) [inline common/source/docs/common-electro-permanent-magnet-V3.rst]: "are" is correct when you refer to plural of something - so "amount of power is" or "amounts of power are". I suggest perhaps "A significant amount of power is"

Comment (ctech4285) [inline common/source/docs/common-electro-permanent-magnet-V3.rst]: What about power, is it not quantized and should be used in plural as well?

**Comment by ctech4285** [comment]
Thank you Hamish all changes done!

**Comment by hamishwillee** [comment]
Thanks @ctech4285 - merged.

> What about power, is it not quantized and should be used in plural as well?

In this case power is actually "singular". If you were to talk about powers you would be referring to multiple types of power (e.g. I have the following superpowers: grammar, spelling, editing).

**Pull request merged**